catalog/message/shape/catalog-request-message-shape.ttl

```diff
@@ -28,9 +28,12 @@ dspace_shapes:
 dspace_shapes:CatalogRequestMessageShape
 	a sh:NodeShape ;
 	sh:targetClass dspace:CatalogRequestMessage ;
+	rdfs:subClassOf dspace_shapes:CatalogShape ;
```

Comment:
CatalogRequestMessage is not a Catalog, so no rdfs:subClassOf of the Shapes. Please remove this line.